tensorboard/components/tf_data_selector/tf-data-selector.html

```diff
@@ -63,13 +65,28 @@
         display: flex;
         flex-direction: column;
         flex-wrap: wrap;
+        width 100%;
       }
 
       #exp-selector {
         display: inline-flex;
         flex-direction: column;
         min-width: 400px;
       }
+
+      tf-data-select-row {
+        margin: 5px 0;
+        max-width: 720px;
+        width: 100%;
+      }
+
+      tf-data-select-row[no-experiment] {
+        margin-left: -8px;
```

Comment:
I assume the negative margin somehow accounts for the checkbox?  But I'm not sure why it's needed if the checkbox disappears when noExperiment is true already...

Reply:
Ah, this is about undoing some padding that paper-input naturally has to align _visual_ left edge with other contents